domain/secretbackend/state/types.go

```diff
@@ -7,20 +7,23 @@ import (
 	"database/sql"
 	"sort"
 
-	"github.com/juju/juju/core/model"
+	coremodel "github.com/juju/juju/core/model"
 	coresecrets "github.com/juju/juju/core/secrets"
 	"github.com/juju/juju/core/watcher"
-	"github.com/juju/juju/domain"
+	"github.com/juju/juju/internal/database"
 )
 
 // Model represents a single subset of a row from the state database's model_metadata table.
 type Model struct {
```

Comment:
This needs to be renamed - it is no longer a pure "Model" construct - it is a ModelSecretBackend or something like that